packages/components/pie-toggle-switch/src/defs.ts

```diff
@@ -1,6 +1,11 @@
 import { RTLComponentProps } from '@justeattakeaway/pie-webc-core';
 
-export interface ToggleSwitchProps extends RTLComponentProps {
+export type LabelProps = {
+    text: string;
+    position: string;
```

Comment:
is okay if we can define the position type as we do with size in the button defs for consistency?

this way we can export the label positions as a const, and can be used in the story.

defs.ts
```
export const labelPositions = ['leading', 'trailing'] as const;

export type LabelProps = {
    text: string;
    position: labelPositions types[number];
}
```

and in the story, we can do 

```
    label: {
            description: 'The label configuration for the toggle switch,
            control: {
                type: 'object',
                defaultValue: {
                    text: 'Label',
                    options: labelPoisitons,
                },
            },
        },

```

Reply:
we probably we wanna export labelPositiosns from index.ts too